pandas/indexes/range.py

```diff
@@ -444,16 +445,19 @@ def join(self, other, how='left', level=None, return_indexers=False):
         how : {'left', 'right', 'inner', 'outer'}
         level : int or level name, default None
         return_indexers : boolean, default False
+        sort : boolean, default False
```

Comment:
same (actually this doc-string should be a shared_doc string if you want to move it), rather than repeating